setup-utils/install-mxnet-osx-python.sh

```diff
@@ -362,19 +360,16 @@ download_mxnet() {
 
 	echo " "
 	echo "MXNET GIT Path = ${MXNET_GITPATH}"
-	#echo "MXNET Tag = ${MXNET_TAG}"
-	#echo "You can set \$MXNET_TAG to the appropriate github repo tag"
-	#echo "If not set, the default value used is the latest release"
-	echo " "
+	echo "MXNET Tag = ${MXNET_TAG}"
+    	echo "You can set \$MXNET_TAG to the appropriate github repo tag"
+    	echo "If not set, the default value used is the latest version on master"
+    	echo " "
```

Comment:
It tells you what version it is about to install without a y/n prompt, and I think it would be good to include that, so users now seeing that they can use an env var to switch versions they won’t have to break out of the script manually to do so (and that’s if they even notice it). Here's an [example prompt](https://github.com/srochel/incubator-mxnet/blob/44813dd69fd6b3e7c9b98b56a55bced7f955fbdd/setup-utils/install-mxnet-osx-python.sh#L129-L135).